docs/admin/system-information.rst

```diff
@@ -1956,57 +1956,67 @@ Snapshots
+|                          | partitioned tables within the    |                              |
+|                          | snapshot.                        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``started``              | The point in time when the       | ``TIMESTAMP WITH TIME ZONE`` |
+|                          | creation of the snapshot         |                              |
+|                          | started. Changes made after      |                              |
+|                          | that are not stored in this      |                              |
+|                          | snapshot.                        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``finished``             | The point in time when the       | ``TIMESTAMP WITH TIME ZONE`` |
+|                          | snapshot creation finished.      |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``state``                | The current state of the         | ``TEXT``                     |
+|                          | snapshot. One of:                |                              |
+|                          | ``IN_PROGRESS``, ``SUCCESS``,    |                              |
+|                          | ``PARTIAL``, or ``FAILED``.      |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``version``              | An internal version this         | ``TEXT``                     |
+|                          | snapshot was created with.       |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``failures``             | A list of failures that occurred | ``ARRAY(TEXT)``              |
+|                          | while taking the snapshot.       |                              |
+|                          | If taking the snapshot was       |                              |
+|                          | successful this is empty.        |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``reason``               | Snapshot failure reason.         | ``TEXT``                     |
+|                          | If taking the snapshot was       |                              |
+|                          | successful this is ``NULL``.     |                              |
++--------------------------+----------------------------------+------------------------------+
+| ``total_shards``         | Total number of shards in the    | ``INTEGER``                  |
+|                          | snapshot.                        |                              |
```

Comment:
While debugging flaky test (see second commit fixing it), I realized that this is actualy total _primary_ shards. 

No mention of replicas in `SnapshotInfo` docs

```
     * Returns total number of shards that were snapshotted; a value of {@code 0} will
     * be returned if {@link #state()} returns {@code null}.
```
Should I explicitly mention `primary` here?

Reply:
> I realized that this is actualy total primary shards.

i.e I checked `sys.shards` - when data nodes is > 1 we have 4 primaries and 4K replicas but totalShards is 4.

Reply:
> Should I explicitly mention primary here?

Probably. A snapshot never contains replicas as these are just copies of the primaries. Anyway, mentioning it explicitly is worth doing.